seahorn/jobs2/priority_queue_top2/aws_priority_queue_top_harness2.c

```diff
@@ -0,0 +1,57 @@
+/**
+ */
+
+#include <aws/common/priority_queue.h>
+#include <priority_queue_helper.h>
+#include <seahorn/seahorn.h>
+#include <utils.h>
+
+/**
+ */
+
+int main(void) {
+  /* data structure */
+  struct aws_priority_queue queue;
+  initialize_priority_queue(&queue);
+
+  /* assumptions */
+  #ifdef __KLEE__
```

Comment:
`ifdef klee` jobs are more complicated. `KLEE` does not have the `reset_modified` keyword or `is_modified`. If putting in `jobs2` we can remove the `KLEE` part -- job is valid for seahorn only